tests/test_pipelines_common.py

```diff
@@ -261,6 +263,27 @@ def __getitem__(self, i):
         for output in text_classifier(dataset):
             self.assertEqual(output, {"label": ANY(str), "score": ANY(float)})
 
+    @require_torch
+    def test_check_task_auto_inference(self):
+        pipeline(model="Narsil/tiny-distilbert-sequence-classification")
+
```

Comment:
Can we check that the result has the right type?

Reply:
Done it, thanks !